src/Microsoft.AspNetCore.HostFiltering/HostFilteringMiddleware.cs

```diff
@@ -72,19 +81,20 @@ public Task Invoke(HttpContext context)
             return _next(context);
         }
 
-        private void EnsureConfigured()
+        private IList<StringSegment> EnsureConfigured()
```

Comment:
Nit: This method is really GetAllowedHosts() now right?

Reply:
I think the PostConfigure logic should be moved into this method to ensure its run 'last'

Reply:
It was David's decision to keep them seperate.